spyder/config/gui.py

```diff
@@ -109,13 +100,6 @@ def set_shortcut(context, name, keystr):
     CONF.set('shortcuts', '%s/%s' % (context, name), keystr)
 
 
-def fixed_shortcut(keystr, parent, action):
-    """Define a fixed shortcut according to a keysequence string"""
-    sc = QShortcut(QKeySequence(keystr), parent, action)
-    sc.setContext(Qt.WidgetWithChildrenShortcut)
-    return sc
-
-
```

Comment:
@rlaverde, please leave this function and add to its docstring this text

    DEPRECATED: This function will be removed in Spyder 4.0